r/vignettes/developers/setup.Rmd

```diff
@@ -47,13 +47,8 @@ recent version of the library without building from source.
 On Linux, you can download a .zip file containing libarrow from the
 nightly repository.
 
-To see what nightlies are available, you can use arrow's (or any other S3 client's) S3 listing functionality to see what is in the bucket `s3://arrow-r-nightly/libarrow/bin`:
+To see what nightlies are available, you can visit the [nightly build server](https://nightlies.apache.org/arrow/r/libarrow/bin). Version numbers in that repository correspond to dates. See the [distro map](https://raw.githubusercontent.com/apache/arrow/master/r/tools/distro-map.csv) for compatibility between binaries and OSs.
```

Comment:
distro-map.csv is still not present in this repo

Reply:
#13241 adds it

Reply:
Is there a good reason that PR is separate from this? Seems odd to merge either of them separately.